### PR TITLE
[WIP] fix selectors, add executable for tests

### PR DESCRIPTION
**Pull request by x0rzkov**
sounds like trending languages have disappeared

### CODE REVIEW

**Comment by andygrunwald** [comment]
Thanks for the PR.
Right now, the tests are failing (seems to be that we need to copy new HTML into the test files) and there is a lot of debugging code in the PR (i assume that this is not needed anymore).

I have not so much time right now to take care of this in detail. Would you mind to continue with this?

**Comment by x0rzkov** [comment]
sounds good, I had to modify some tests as the all language option is not available

**Comment by x0rzkov** [comment]
Any updates on this PR ?

**Comment by x0rzkov** [comment]
Shall I close this PR ?

File: trending_test.go [hunk 1]
```diff
@@ -188,12 +188,13 @@ func TestGetTrendingLanguages(t *testing.T) {
 	// Might be dirty, but hey ...
 	// a) it works
 	// b) how high is the chance that All Languages + Unknown language change? ;)
+	/*
 	allLanguages := languages[0]
-	if allLanguages.Name != "All languages" {
-		t.Errorf("GetLanguages returned %+v, want %+v", allLanguages.Name, "All languages")
+	if allLanguages.Name != "C" {
```
Comment (andygrunwald) [inline trending_test.go:193]: What is the reason to deactivate this test?

File: trending.go [hunk 2]
```diff
@@ -343,36 +357,40 @@ func (t *Trending) GetDevelopers(time, language string) ([]Developer, error) {
...
 func (t *Trending) newDeveloper(name, fullName string, linkURL, avatarURL *url.URL) Developer {
-	return Developer{
+	d := Developer{
 		ID:          t.getUserIDBasedOnAvatarURL(avatarURL),
 		DisplayName: name,
 		FullName:    fullName,
 		URL:         linkURL,
 		Avatar:      avatarURL,
 	}
+	// pp.Println("Developer: ", d)
```
Comment (andygrunwald) [inline trending.go:392]: Would be good practice to clean up the debugging statements here.

File: .gitignore [hunk 3]
```diff
@@ -28,4 +28,5 @@ cover*
 .vscode/
 
 # Build folder
-build/
\ No newline at end of file
+build/
+cmd/
```
Comment (andygrunwald) [inline .gitignore:32]: What is the reason to exclude the cmd folder?

**Comment by andygrunwald** [comment]
Thanks for the PR. We adjusted a fore more things in https://github.com/andygrunwald/go-trending/pull/21

**Pull request closed without merging**